code/modules/clothing/suits/jobs.dm

```diff
@@ -51,7 +51,7 @@
 	inhand_icon_state = "apronchef"
 	blood_overlay_type = "armor"
 	body_parts_covered = CHEST|GROIN
-	allowed = list(/obj/item/kitchen, /obj/item/knife/kitchen)
+	allowed = list(/obj/item/kitchen, /obj/item/knife/kitchen, 	/obj/item/storage/bag/tray)
```

Comment:
2 spaces

Reply:
Fixed, but will post it later as I have no Ethernet currently and writing from phone